chapter_18_spiking_custom_auth.asciidoc

```diff
@@ -53,7 +54,7 @@ It's nice and straightforward, and I'll leave it to you to discover on your own.
 What authentication system could we use to avoid storing passwords ourselves?
 Oauth?  Openid?  "Login with Facebook"?   Ugh.
 For me those all have unacceptable creepy overtones;
-why should Google or Facebook know what sites you're logging into and when?
+why should Google or Facebook know what sites ur logging into and when?
```

Comment:
hey @sgrey513 i really like apostrophes pls can we not say ur

Reply:
Oh... fine

Reply:
Hello again